### DISCUSSION TITLE
Custom delimiter for use in Laravel Blade file?

### DISCUSSION THREAD

**Discussion by rguttersohn** [vuejs/petite-vue#141] [General]
Anyone using Petite Vue with Laravel find a custom delimiter to use inside a Blade file? I am hoping to avoid the use of the Blade verbatim directive throughout my HTML when integrating Petite Vue. 

Let me know if anyone has figured this out.

**Comment by halivert** [comment]
Yes, you can define it on creation
https://github.com/vuejs/petite-vue#custom-delimiters-03

**Reply by rguttersohn** [reply]
Actually I am just realizing you Blade allows you to add an @ in front of Vue's double brackets, thus removing the need for wrapping lines of code in the verbatim directive.

**Reply by halivert** [reply]
Yes there are different solutions, I also use the one that you said. 😄
